phys/module_mp_thompson.F

```diff
@@ -3579,10 +3581,10 @@ subroutine qr_acr_qg
 
       good = 0
       IF ( wrf_dm_on_monitor() ) THEN
-        INQUIRE(FILE="qr_acr_qgV2.dat",EXIST=lexist)
+        INQUIRE(FILE="qr_acr_qgV3.dat",EXIST=lexist)
```

Comment:
@gthompsnWRF 
Greg,
Not *this* time, but next release.
How about in the module_mp_thompson.F, above the contains statement, that you have a few private character strings, the names of your files. Then when required, we change the file names only in a single place. As everything is in the module, there would be no need to pass the character strings around (as they would have global scope within the module). Since we are changing file names every year or so, this seems to be a clean way to move forward.

Reply:
Yes, that would be a great item to check off the list, because the name appears in multiple places and we can resolve it quickly with a module-wide variable for each of the lookup tables.

FYI:  I will tackle this item as part of my next code update, which is far larger, because I am introducing 3-moment graupel-hail addition to MP scheme.  So it will be logical to incorporate that as well, because this file will need another updated version number.